### PR TITLE
fix(DB/Creature): Laris Geardawdle (9616) missing text

### PR DESCRIPTION
**Pull request by r0m1ntik**
## Changes Proposed:
-  Add condition for dialog with npc
-  Add text in gossip_menu

## Issues Addressed:
- Closes [4131](https://github.com/azerothcore/azerothcore-wotlk/issues/4133)

## Tests Performed:
Снимок экрана 2021-01-02 в 16 22 41
Снимок экрана 2021-01-02 в 16 31 43
Снимок экрана 2021-01-02 в 16 31 46
Снимок экрана 2021-01-02 в 16 31 49
Снимок экрана 2021-01-02 в 16 31 52

## How to Test the Changes:

1) Apply SQL
2) .go cr id 9616
3) .q add 4512 and .q complete 4512
4) .q add 4513
5) **View dialog**

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
good job @r0m1ntik !

**Comment by JamesGent** [comment]
Tested and works fine on my end.

**Comment by FrancescoBorzi** [comment]
thanks @JamesGent for confirming the solution!

File: data/sql/updates/pending_db_world/rev_1609601212878725000.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    (2406, 3100),
+    (2407, 3101),
+    (2408, 3102);
+DELETE FROM `conditions` WHERE `SourceGroup` = 2405;
+INSERT INTO `conditions` VALUES
+    (14, 2405, 3097, 0, 0, 8, 0, 4512, 0, 0, 0, 0, 0, '', 'Show gossip text 3099 if quest \'A Little Slime Goes a Long Way (Part 1)\' is rewarded'),
+    (14, 2405, 3097, 0, 0, 9, 0, 4513, 0, 0, 1, 0, 0, '', 'Show gossip text 3099 if quest \'A Little Slime Goes a Long Way (Part 2)\' is not taken'),
+    (14, 2405, 3099, 0, 0, 9, 0, 4513, 0, 0, 0, 0, 0, '', 'Show gossip text 3099 if quest \'A Little Slime Goes a Long Way (Part 2)\' is taken'),
+    (14, 2405, 3098, 0, 0, 8, 0, 4513, 0, 0, 0, 0, 0, '', 'Show gossip text 3098 if quest \'A Little Slime Goes a Long Way (Part 2)\' is rewarded'),
+    (15, 2405, 0, 0, 0, 9, 0, 4513, 0, 0, 0, 0, 0, '', 'Show gossip option if quest \'A Little Slime Goes a Long Way (Part 2)\' is taken');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1609601212878725000.sql:17]:
```suggestion
INSERT INTO `gossip_menu` (`MenuID`, `TextID`) VALUES
(2405, 3077),
(2405, 3097),
(2405, 3098),
(2405, 3099),
(2406, 3100),
(2407, 3101),
(2408, 3102);
DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` = 14 AND `SourceGroup` = 2405;
INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
(14, 2405, 3097, 0, 0, 8, 0, 4512, 0, 0, 0, 0, 0, '', 'Show gossip text 3099 if quest \'A Little Slime Goes a Long Way (Part 1)\' is rewarded'),
(14, 2405, 3097, 0, 0, 9, 0, 4513, 0, 0, 1, 0, 0, '', 'Show gossip text 3099 if quest \'A Little Slime Goes a Long Way (Part 2)\' is not taken'),
(14, 2405, 3099, 0, 0, 9, 0, 4513, 0, 0, 0, 0, 0, '', 'Show gossip text 3099 if quest \'A Little Slime Goes a Long Way (Part 2)\' is taken'),
(14, 2405, 3098, 0, 0, 8, 0, 4513, 0, 0, 0, 0, 0, '', 'Show gossip text 3098 if quest \'A Little Slime Goes a Long Way (Part 2)\' is rewarded'),
(15, 2405, 0, 0, 0, 9, 0, 4513, 0, 0, 0, 0, 0, '', 'Show gossip option if quest \'A Little Slime Goes a Long Way (Part 2)\' is taken');
```

**Review by Kitzunu — CHANGES_REQUESTED** [review]
For good measures incase the tables would ever change.

**Pull request merged**